java/src/jmri/jmrit/vsdecoder/swing/ManageLocationsFrame.java

```diff
@@ -354,6 +355,15 @@ public LocationTableModel(Object[][] dataMap) {
             HashMap<String, PhysicalLocation> retv = new HashMap<String, PhysicalLocation>();
             for (Object[] row : rowData) {
                 if ((Boolean) row[1]) {
+                    if (row[2] == null) { 
+                        row[2] = 0.0f;
+                    }
+                    if (row[3] == null) { 
+                        row[3] = 0.0f;
+                    }
+                    if (row[4] == null) { 
+                        row[4] = 0.0f;
```

Comment:
Not a problem, just curious, but you check elements 2, 3 and 4 for "null" here, then reference 2, 3, 4 and 5.  Is there some reason to believe element 5 can't be null? It's a Boolean type, but those can also be null.